install.sh

```diff
@@ -315,13 +300,49 @@ unzip() {
     local _file="$1"
     local _dir="$2"
 
-    tar -xzvf "$_file" -C "$_dir"
+    tar -xzf "$_file" -C "$_dir"
```

Comment:
This still doesn't work on windows. it should be fixed or have an issue opened to track.

Reply:
there is an issue open